guidelines/php/drupal/README.md

```diff
@@ -0,0 +1,102 @@
+Writing PHP in Drupal 8
+=======================
+
+We follow the [Drupal Coding standards](https://www.drupal.org/docs/develop/standards).
+For our Drupal 8 projects we have a separate [How-do-we-drupal-article](https://github.com/gridonic/drupal8).
+
+When you read carefully our global guidelines you find the line:
+
+> We use 4 spaces for indentation
+
+In our drupal projects we will ignore the 4 spaces in all php- & twig-files and we use just 2 spaces for indentation.
+
+## Exceptions
+
+During the time of developing Drupal-projects we have noticed that not all rules are really suitable.
```

Comment:
During the time of developing Drupal-projects we have noticed that not all rules are really suitable.
+So we have allowed some exceptions we want to list here:

We adhere to most rules for Drupal project, but add some exceptions where we disagree or find the original Drupal rules not suitable for our use case(s). These exceptions are listed below.

Reply:
Thx for the improvements! I fixed it.